### ISSUE TITLE
Abbreviations should be enclosed in <abbr> element and have their title attribute filled in

### ISSUE THREAD

**Issue by anselmbradford** [codeforamerica/ohana-web-search#102]
Abbreviations should be enclosed in the <abbr> element and have their title attribute properly filled out with the abbreviation meaning. 

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr
